lahja/base.py

```diff
@@ -70,7 +82,7 @@ def serve(cls, config: ConnectionConfig) -> AsyncContextManager["BaseEndpoint"]:
             # server stopped
 
         """
-        pass
+        ...
```

Comment:
@carver @cburgdorf what do you think about this convention.  It feels like it make `abstractmethod`s more visually distinct.

Reply:
I'm happy with it :+1: Although if we start using `autoflake` maybe it will be moot because either one would be stripped out.

Reply:
Looks like `autoflake` leaves `...` in place so maybe that's an argument to use it.

Reply:
Interesting :+1: